### ISSUE TITLE
Title scans but no files are created

### ISSUE THREAD

**Issue by mastressalita** [Cortys/comic-backup#88]
Hello, I'm having issues backing up one comic in my collection, but it is working fine on all the others in my Comixology library. The only one it is not working on is a manga called "Kashimashi ~ Girl Meets Girl ~ Vol. 1". I've successfully extracted files with the exact same settings all the other manga in my account, but this one goes through the "Scanning" process on another tab, but doesn't create the image files, and I'm perplexed what is going on. I'm curious if the tildes in the title could be causing an issue and not allowing the creation of the files? (It's a shot in the dark guess, I'm totally out of ideas here). I've tried changing the settings, but nothing has worked, only on this one title. Any help would be appreciated.

**Comment by Cortys** [comment]
Thanks for reporting this bug! I just pushed a fix for it to master.

Please let me know whether this solves the problem.

**Comment by mastressalita** [comment]
Thanks for looking into this. Now it just says "Preparing" and doesn't move into the scanning phase. Likewise this happens when I try different titles, as well.

**Comment by Cortys** [comment]
I see. Since I cannot reproduce the issue, please check whether any errors are logged.

There are three main locations to check:
* **If** there are errors in the background scanning script, there should be a red "Errors" button next to the extension in chrome://extensions/. Click on that button and screenshot the output.
* On the Comixology page, open the error log by opening the dev console (Ctrl+Shift+i). Also screenshot the log there.
* Lastly, assuming that the comic tab opens in the background, please also post the output in the dev console of that tab too (also via Ctrl+Shift+i).

**Comment by mastressalita** [comment]
I didn't have any errors, it was behaving similar to the report in this section for "Download Doesn't Start, Workaround Exists" which I wasn't having before. I tried the "workaround" which did work. In any event, I removed the addon again, rebooted the computer, re-added it, set back up my settings, and have tried it a second time. No "Preparing" hang up this time! It has worked on the "Kashimashi ~ Girl Meets Girl ~" file that was giving me issues before, and I tried it again on an old file (just for kicks) and that is also working. Thank you so much! It was really making me scratch my head why only one of my manga wasn't working, it seems it was the tilde after all!
